src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/CustomCaConfigurationTest.java

```diff
@@ -162,41 +162,31 @@ private void givenCDAWithCustomCertificateAuthority(URI privateKeyUri, URI certi
         kernel.getContext().waitForPublishQueueToClear();
     }
 
+    private GetCertificateRequest buildCertificateUpdateRequest(
```

Comment:
This were just tweaks to the test so I could reuse the subscription part and simulate other components subscribing to CDA.